news/2024/2024-12-21-new-featured-artist-anamanaguchi.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: post
+title: "New Featured Artist: Anamanaguchi"
+date: 2024-12-21 20:00:00 +0000
+---
+
+**Anamanaguchi** rounds out the year as our final Featured Artist of 2024!
+
+![](https://assets.ppy.sh/artists/454/header.jpg)
+
+For two decades and counting, **Anamanaguchi** has been the powerhouse behind combined chiptune/bitpop and traditional pop/rock music. From their own original releases to the soundtrack for multiple adaptations of *Scott Pilgrim vs. the World*, **Anamanaguchi** has made an unforgettable name for themselves in all kinds of aural circles (...the most important of those being osu!, of course).
+
+Through old-school mapper [Sushi](https://osu.ppy.sh/users/43108), **Anamanaguchi** made their first circle-paired appearance via [Helix Nebula](https://osu.ppy.sh/beatmapsets/10517) in 2009. What followed was an onslaught of maps year after year, highlighting the band's massive discography with beatmaps like [Blackout City](https://osu.ppy.sh/beatmapsets/67226) from *Dawn Metropolis*, [SPF 420](https://osu.ppy.sh/beatmapsets/109024) from *Endless Fantasy*, and most recently [Miku](https://osu.ppy.sh/beatmapsets/1839623), which reached the Ranked category only last year.
+
+Enough words. There are **9** tracks on [**Anamanaguchi**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/454), each of whose greatness speaks for itself.
+
+For those who prefer completed maps over templates, take a glance at what's already here in osu!:
+
+<div align="center">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/artists/454/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### [Anamanaguchi - STILL SPLODIN' THO (Interlude)](https://assets.ppy.sh/artists/454/Endless%20Fantasy/Anamanaguchi%20-%20STILL%20SPLODIN%27%20THO%20(Interlude).osz)
+
+It might say something about the osu! community to know that [this 1 minute map](https://osu.ppy.sh/beatmapsets/104151) hosted by [Liiraye](https://osu.ppy.sh/users/1280641) is the most played **Anamanaguchi** map on osu!.
```

Comment:
```suggestion
It might say something about the osu! community to know that [this 1-minute map](https://osu.ppy.sh/beatmapsets/104151) hosted by [Liiraye](https://osu.ppy.sh/users/1280641) is the most played **Anamanaguchi** map on osu!.
```